tests/integration_tests/test_shap.py

```diff
@@ -194,7 +211,7 @@ def objective(trial: Trial) -> Tuple[float, float]:
         study.optimize(objective, n_trials=3)
 
         with pytest.raises(ValueError):
-            ShapleyImportanceEvaluator().evaluate(study)
+            get_param_importances(study, evaluator=ShapleyImportanceEvaluator())
```

Comment:
I'd say it's better to focus the behavior of `ShapleyImportanceEvaluator().evaluate(study)` in this test file. So how about calling `ShapleyImportanceEvaluator().evaluate(study)` directly?

Reply:
`ShapleyImportanceEvaluator().evaluate` does not handle errors anymore, so we cannot directly test it for invalid cases.
Maybe we do not need these tests because the error-handling behavior is already tested in `tests/importance_tests/test_init.py`.

Reply:
Agree 👍  I think we can simply remove these tests.